warp10/src/main/sh/warp10.sh

```diff
@@ -17,506 +17,299 @@
-  sed -i -e 's|^ExecStop=.*|ExecStop='${WARP10_HOME_ESCAPED}'/bin/warp10.sh stop|' ${WARP10_HOME}/bin/warp10.service
+distConf() {
+  echo "Initialize Warp 10 distributed configuration"
+  init
+  postInit
+}
 
-  sed -i${SED_SUFFIX} -e 's|warpLog\.File =.*|warpLog.File = '${WARP10_HOME_ESCAPED}'/logs/warp10.log|' ${WARP10_HOME}/etc/log4j.properties
-  sed -i${SED_SUFFIX} -e 's|warpscriptLog\.File =.*|warpscriptLog.File = '${WARP10_HOME_ESCAPED}'/logs/warpscript.out|' ${WARP10_HOME}/etc/log4j.properties
-  rm ${WARP10_HOME}/etc/log4j.properties${SED_SUFFIX}
+leveldbConf() {
+  echo "Initialize Warp 10 standalone configuration"
+  init
 
+  echo "standalone.home = ${WARP10_HOME_ESCAPED}" >>"${WARP10_CONFIG_DIR}/99-init.conf"
+  echo "backend = leveldb" >>"${WARP10_CONFIG_DIR}/99-init.conf"
+  mv "${WARP10_CONFIG_DIR}/10-fdb.conf" "${WARP10_CONFIG_DIR}/10-fdb.conf.DISABLE"
   getConfigFiles
 
-  # Generate secrets
-  chown -R ${WARP10_USER}:${WARP10_GROUP} ${WARP10_CONFIG_DIR}
-  su ${WARP10_USER} -c "${JAVACMD} -cp ${WARP10_JAR} -Dfile.encoding=UTF-8 io.warp10.GenerateCryptoKey ${CONFIG_FILES}"
 
-  # Edit the warp10-tokengen.mc2 to use or not the secret
-  secret=`su ${WARP10_USER} -c "${JAVACMD} -cp ${WARP10_JAR} -Dlog4j.configuration=file:${LOG4J_CONF} -Dfile.encoding=UTF-8 io.warp10.WarpConfig ${CONFIG_FILES} . 'token.secret' | grep -e '^@CONF@ ' | sed -e 's/^@CONF@ //' | grep 'token.secret' | sed -e 's/^.*=//'"`
-  if [[ "${secret}"  != "null" ]]; then
-    sed -i${SED_SUFFIX} -e "s|^{{secret}}|'"${secret}"'|" ${WARP10_HOME}/templates/warp10-tokengen.mc2
-  else
-    sed -i${SED_SUFFIX} -e "s|^{{secret}}||" ${WARP10_HOME}/templates/warp10-tokengen.mc2
+  ##
+  ##  Init LevelDB
+  ##
+  LEVELDB_HOME="${WARP10_HOME_ESCAPED}/leveldb"
+  echo "Initialize LevelDB"
+  if ! mkdir -p "${LEVELDB_HOME}/snapshots"; then
+    die "ERROR: ${LEVELDB_HOME} creation failed"
   fi
-  rm ${WARP10_HOME}/templates/warp10-tokengen.mc2${SED_SUFFIX}
 
-  # Generate read/write tokens valid for a period of 100 years. We use 'io.warp10.bootstrap' as application name.
-  su ${WARP10_USER} -c "${JAVACMD} -cp ${WARP10_JAR} -Dlog4j.configuration=file:${LOG4J_CONF} -Dfile.encoding=UTF-8 io.warp10.TokenGen ${CONFIG_FILES} ${WARP10_HOME}/templates/warp10-tokengen.mc2" >  ${WARP10_HOME}/etc/initial.tokens
-  sed -i${SED_SUFFIX} 's/^.\{1\}//;$ s/.$//' ${WARP10_HOME}/etc/initial.tokens # Remove first and last character
-  rm "${WARP10_HOME}/etc/initial.tokens${SED_SUFFIX}"
+  ${JAVACMD} -cp "${WARP10_JAR}" io.warp10.standalone.WarpInit "${LEVELDB_HOME}" >>"${WARP10_HOME}/logs/warp10.log" 2>&1
 
-  echo "Warp 10 config has been generated here: ${WARP10_CONFIG_DIR}"
+  postInit
+}
 
-  echo "You can now configure the initial and maximum amount of RAM allocated to Warp 10. Edit ${WARP10_HOME}/bin/warp10-env.sh, and look for WARP10_HEAP and WARP10_HEAP_MAX variables."
+fdbConf() {
```

Comment:
rename it standalonePlus so as to not think it is used when initializing a distributed version.

Reply:
done